modules/php/Cards/GeneralStore.php

```diff
@@ -25,13 +26,14 @@ public function play($player, $args)
     parent::play($player, $args);
     $players = Players::getLivingPlayersStartingWith($player);
     Log::addAction('selection', ['players' => $players, 'src' => $this->name, 'card' => 1]);
-    Cards::createSelection(count($players));
+    $cards = Cards::createSelection(count($players));
+    $player->prepareSelection($this, $cards, $players);
     return 'selection';
   }
 
   public function react($card, $player)
   {
-    Cards::moveCard($card, 'hand', $player->getId());
+    Cards::move($card->getId(), 'hand', $player->getId());
     Notifications::chooseCard($player, $card);
     return null;
```

Comment:
Same here

Reply:
Removed return from all cards, hope this will still work :)